### DISCUSSION TITLE
Getting 502 bad gateway error

### DISCUSSION THREAD

**Discussion by Vishu1701** [community/community#39684] [GitHub Education]
I recently tried to get GitHub Student Developer Pack and submitted all my college details and proof as well but when I submitted then after 2 mins, I got a 502 error and am now stuck.

**Comment by david-engelmann** [comment]
[You can reach out to Github Education support here](https://support.github.com/contact/education)! They can also assist you if you contact them through support@github.com!

**Comment by samus-aran** [comment]
Hi @Vishu1701 👋 

Apologies that you have received that 502 error. Is this still occurring? You may need to re-submit your evidence if there was a problem during the form submission.
